src/components/task-plan/reading.cjsx

```diff
@@ -82,91 +99,99 @@ SelectTopics = React.createClass
       </div>
     </BS.Modal>
 
+ReadingFooter = React.createClass
+  render: ->
+    classes = []
+    classes.push('disabled') unless @props.enabled
```

Comment:
Yep, agreed. and then moving forward `props` are always defined in `proptypes` first.